Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SD-3-B.yml

```diff
@@ -16,23 +16,24 @@ id:
 
 #The title of this activity
 title:
-  Files and repositories are checked periodically for secrets that should be protected.
+  Favor run-time exposure to secrets and improve lifecycle management.
 
 #Describe the benefit that is achieved by implementing this activity
 benefit: 
-  Risk of leaking production secrets is mitigated by removing all manual interactions and regular regeneration.
+  Risk of leaking production secrets is reduced by limiting access only during run-time and by regenerating regularly.
```

Comment:
Sounds a bit weird to me. The idea was that optimally, nobody should have to touch any secrets manually at all?

Reply:
Reworded a bit.